docs/guide/installation.md

```diff
@@ -248,17 +248,15 @@ cp <lnbits_repo>/.env.example .env
 
 and change the configuration in `.env` as required.
 
-Then create the data directory for the user ID 1000, which is the user that runs the lnbits within the docker container.
-
+Then create the data directory
 ```
 mkdir data
-sudo chown 1000:1000 ./data/
 ```
 
 Then the image can be run as:
 
 ```
-docker run --detach --publish 5000:5000 --name lnbits --volume ${PWD}/.env:/app/.env --volume ${PWD}/data/:/app/data lnbits
+docker run --detach --publish 5000:5000 --name lnbits-legend --volume ${PWD}/.env:/app/.env --volume ${PWD}/data/:/app/data lnbits-legend
```

Comment:
nitpick: add one environment variable mapping (to serve as an example)

Reply:
what i though i did that :D? thanks

Reply:
what do you mean?

Reply:
Something like:
- `-e LNBITS_BACKEND_WALLET_CLASS='FakeWallet'`
or
- `-e LNBITS_DATABASE_URL="postgres://lnbits:password123@localhost:5432/lnbits_db"`

It can be separate comment, just for providing an example.